Woodoku/src/Entity/WoodokuBoard.py

```diff
@@ -12,17 +14,36 @@ class WoodokuBoardRepresentation:
     The first block on the bottom row is (n - 1, 0).
     The bottom-right block's coordinate is (n - 1, n - 1).
     """
-    def __init__(self):
-        pass
+    board: ndarray
+    length: int
+
+    def __init__(self, n: int):
+        self.length = n
+        self.board = np.zeros((self.length, self.length))
 
     def __str__(self):
-        pass
+        # reformat = np.reshape(self.board, self.length * self.length)
+        return np.array2string(self.board)
```

Comment:
this is a little too primitive and not quite visualizable with just 1 and 0

Reply:
This method was reserved to use in `CommandLineUI`. We might need a more detailed design on the print out format.